src/Spice86.Core/Emulator/InterruptHandlers/Input/Keyboard/KeyboardInt16Handler.cs

```diff
@@ -137,6 +148,22 @@ public void FlushKeyboardBuffer() {
     /// <inheritdoc/>
     public override void Run() {
         byte operation = State.AH;
-        Run(operation);
+
+        if (HasRunnable(operation)) {
+            Run(operation);
+        } else {
+            HandleUndocumentedInterrupt(operation);
+        }
+    }
+
+    private void HandleUndocumentedInterrupt(byte operation) {
```

Comment:
This hides unimplemented interrupts with logs, I'd rather follow @kevinferrare design and remove this (and changes in the calling code) to let the exception crash the emulator on unimplemented behavior, or implement the operation.